org.opennumismat.OpenNumismat.yml

```diff
@@ -0,0 +1,36 @@
+app-id: org.opennumismat.OpenNumismat
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: open-numismat
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '5.15-21.08'
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=host
```

Comment:
that's a very broad permission for managing coins

Reply:
`- --filesystem=host` is what I was refering to.

Reply:
Users need access to USB-devices (for import mobile version) and dropbox/google disk folders for backups. If mounting them to home folder is a good practice, then such permission are of course not required.

Reply:
Yeah. home, /media and /run/media would be better. `/var/tmp` is writable by default, so yes, you can set TMPDIR to it.